### PR TITLE
feat: add usgov support

### PR DESCRIPTION
**Pull request by jmolnar-zscaler**

### CODE REVIEW

**Review by anichari — APPROVED** [review]
LGTM

File: examples/base_1cc/variables.tf [hunk 1]
```diff
@@ -249,3 +249,9 @@ variable "support_access_enabled" {
   description = "If Network Security Group is being configured, enable a specific outbound rule for Cloud Connector to be able to establish connectivity for Zscaler support access. Default is true"
   default     = true
 }
+
+variable "zssupport_server" {
+  type        = string
+  description = "destination IP address of Zscaler Support access server. IP resolution of remotesupport.<zscaler_customer_cloud>.net"
+  default     = "199.168.148.101" #for commercial clouds
```
Comment (rhartnett-zscaler) [inline examples/base_1cc/variables.tf:256]: Is it expected that this IP is different for gov clouds?

File: modules/terraform-zscc-bastion-azure/variables.tf [hunk 2]
```diff
@@ -49,7 +49,10 @@ variable "instance_size" {
 }
 
 locals {
-  unsupported_regions = ["chinanorth", "chinaeast", "china east", "china north"]
+  unsupported_regions          = ["chinanorth", "chinaeast", "china east", "china north", "usgovirginia"]
```
Comment (rhartnett-zscaler) [inline modules/terraform-zscc-bastion-azure/variables.tf:52]: Do we use this local variable any more?

File: modules/terraform-zscc-workload-azure/variables.tf [hunk 3]
```diff
@@ -49,7 +49,10 @@ variable "instance_size" {
 }
 
 locals {
-  unsupported_regions = ["chinanorth", "chinaeast", "china east", "china north"]
+  unsupported_regions          = ["chinanorth", "chinaeast", "china east", "china north", "usgovirginia"]
```
Comment (rhartnett-zscaler) [inline modules/terraform-zscc-workload-azure/variables.tf:52]: same question here

File: examples/base_1cc/variables.tf (hunk 1 above)
Comment (jmolnar-zscaler) [reply examples/base_1cc/variables.tf:256]: Correct. remotesupport.<zscaler_customer_cloud>.net <<< 199.168.148.101 is the resolved IP for all commercial clouds.

This is specified in the variable description. ZSEC handles this automatically doing an fqdn lookup by cloud so if gov will update the nsg rule IP

else
                    echo "Resolving remotesupport.$zscaler_cloud to IP for Network Security Group rule..."
                    support_server_ip=$(dig +short remotesupport.$zscaler_cloud)
                    echo "${GREEN}Outbound rule permitting TCP/12002 access to $support_server_ip will be created${RESET}"
                    echo "export TF_VAR_zssupport_server='$support_server_ip'" >> .zsecrc
                fi

File: modules/terraform-zscc-bastion-azure/variables.tf (hunk 2 above)
Comment (jmolnar-zscaler) [reply modules/terraform-zscc-bastion-azure/variables.tf:52]: yeah, i left this around mainly for just for china as it has some nuance issues with os_disk types in my testing

storage_account_type = contains(local.unsupported_regions, lower(var.location)) ? "Standard_LRS" : "Premium_LRS"

**Review by rhartnett-zscaler — APPROVED** [review]

**Pull request merged**